src/main/java/org/worldcubeassociation/db/Persons.java

```diff
@@ -38,4 +38,14 @@ public Person findById(String aId) {
         return existingPersons;
     }
 
+    public boolean existNameAndCountry(String aName, String aCountry) {
```

Comment:
At first I used that existing method (that seems not to have use elsewhere in the code, btw) but it doesn't make a perfect match for names (uses `endsWith()` instead) and besides, is less efficient as it doesn't stop at the first coincidence.  But if you still want me to use the existing method, it's fine for me.